ReactWindows/Playground.Net46/Playground.Net46.csproj

```diff
@@ -160,11 +157,23 @@
       <Project>{22cbff9c-fe36-43e8-a246-266c7635e662}</Project>
       <Name>ReactNative.Net46</Name>
     </ProjectReference>
+    <ProjectReference Include="..\ReactNative.Net46\ReactNative.Net46.csproj">
```

Comment:
Adding a project reference to ReactNative.Net46... Previously, it was a manual reference, which caused the x86 build to fail.